odoo/fields.py

```diff
@@ -220,6 +220,18 @@ def _search_upper(self, operator, value):
         computation gives the expected value. Note that a computed field without
         an inverse method is readonly by default.
 
+        .. warning::
+
+            While it is possible to use the same compute method for multiple
+            fields, it is not recommended to do the same for the inverse
+            method. This is because during the computation of the inverse,
+            **all** fields that use said inverse will be marked as protected,
+            meaning that they can't be computed, and if there's no value for
+            them inside the record cache the ORM will simply return a default
+            value of `False` for these fields, so if these fields are accessed
+            inside the inverse function, they most likely will return `False`
+            instead of the correct value and the inverse method won't work.
```

Comment:
7 lines for one sentence ? The explanation could be split in 2/3 sentences ;).

```suggestion
            method. 
            
            During the computation of the inverse, **all** fields that use 
            said inverse are protected, meaning that they can't be computed,
            even if their value is not in the cache.

            If any of those field is accessed and its value is not in cache, 
            the ORM will simply return a default value of `False` for these fields.
            This means that the value of the other inverse fields 
            (than the one triggering the inverse method) may not give their correct
            value and this will probably break the expected behavior of the inverse method.
```

Reply:
Indeed, thanks